virttest/qemu_vm.py

```diff
@@ -123,6 +123,10 @@ class VM(virt_vm.BaseVM):
     # By default we inherit all timeouts from the base VM class except...
     CLOSE_SESSION_TIMEOUT = 30
     MIGRATE_TIMEOUT = 2000
+    #: By default translate standard and experimental (prefixed with x-)
+    #: options according to supported parameters/capabilities. To turn this
+    #: off enable this option
+    DISABLE_AUTO_X_MIG_OPTS = False
```

Comment:
why do we hardcode the option ? should this be a param instead.

Reply:
Well it was suggested to me in the request to make it tweakable and I do agree it's very rarely to be modified (if ever) so not polluting params makes sense in this case to me. I don't really care, I'd ideally drop this commit and had this always on.